signinapp/events.py

```diff
@@ -159,23 +186,57 @@ def list_upcoming_events():
     return render_template("events.html.jinja2", prefix="Upcoming ", events=events)
 
 
+@events.route("/events/open")
+def list_open_events():
+    events: list[Event] = db.session.execute(
```

Comment:
I'm curious if there's a difference between `db.session.execute(...).scalars()` and `db.session.scalars(...)`

Reply:
They're equivalent, but `db.session.scalars(...)` is obviously shorter. I've updated all the calls in events.py (didn't see any others) to use this form.